tests/unit/parsers/test.manifestjson.js

```diff
@@ -2736,4 +2738,180 @@ describe('ManifestJSONParser', () => {
       return testHomepageUrl('http://test.org', true);
     });
   });
+
+  describe('@mdn/browser-compat-data', () => {
```

Comment:
Nit, we are checking only the manifest keys compat data in these test at the moment, maybe we can change the description to `'@mdn/browser-compat-data: manifest properties'` or something like that.